### ISSUE TITLE
Host pages - inconsistent photo, description layout

### ISSUE THREAD

**Issue by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#143]
Firstly - the Hosts pages are SOOOOOO much better! Thank you!!

There is an inconsistency between photo cropping and description placement (along w a number of minor related items)

For instance, compare the Hosts pages for Wes, Chris, Brent:

On an Episode page - looks great! ✅ 
![image](https://user-images.githubusercontent.com/7018928/180277071-db902339-0800-44ba-b361-4a41c06d5047.png)

Chris
![image](https://user-images.githubusercontent.com/7018928/180277149-efee89c4-bc5f-4e71-9e0c-603433e7cd45.png)

Wes
![image](https://user-images.githubusercontent.com/7018928/180277202-67f8bcb2-51c4-49e5-a1f3-c9a0c3c98644.png)

Brent
![image](https://user-images.githubusercontent.com/7018928/180277263-299c1b6c-a23c-4983-abfc-95026a437f7a.png)

**Comment by harriseldon** [comment]
I did a quick experiment in developer mode. I took Brent's bio and applied it to Wes's and Chris's page. The ratio of picture to content matched and the 1/3:2/3 properly spread out to take up the entire card. I wonder if the is-one-third-tablet and is-two-thirds-tablet do not apply correctly depending on the amount of text.

**Comment by simoncaron** [comment]
Looks like it's also an issue on mobile:

![image](https://user-images.githubusercontent.com/8635747/181355408-2f51ee4b-1176-469c-845d-b58a7d9466d4.png)

**Comment by gerbrent** [comment]
As a (thus biases) photographer, can I just state my opinion that the photograph should *never* have it's height-width ratios modified? I mean... we're cropping people out of their own photographs! These ratios should be locked, and the photograph should never be cropped, only scaled.

**Comment by CGBassPlayer** [comment]
> These ratios should be locked, and the photograph should never be cropped, only scaled.

The only thing with that is we loose uniformity in the cards. I think for this page, it will be fine. But we should be careful when we have multiple cards on a screen. Having card images different ratios could really ruin the look of a site.

Just throwing a different perspective on this

**Comment by gerbrent** [comment]
that's a very good point, and totally agree. Open to suggestions on how to solve this one. Anything better than the current situation is welcomed ; )

**Comment by gerbrent** [comment]
Perhaps an acceptable compromise could be a square crop viewport with one side always maximizing the photograph's dimensions?

i.e. given a 500wx750h photograph, it gets cropped to 500x500 or the equivalent ratio. 
i.e. only one dimension is ever cropped.

**Comment by Lord0fBytes** [comment]
OK, I fixed the horizontal (desktop viewport) layout issues. There was missing line in the CSS. 

I am still working on the mobile version though. I believe the reason this is happening is because we strode away from the core design of what Bulma uses for the cards a little bit. Basically we are using an empty div with a background image for the picture. So when it goes below a minimum height it is being set to the `min-height` variable which is hard coded for `215px`. I should have a fix for this soon.

**Comment by Lord0fBytes** [comment]
OK, so I fixed this but I also modified the layout a little bit. 
Let me know what you think but I adjusted the layout to be 2/3's page for 'host card' and 1/3 for shows. This helps with the aspect ratio of the photos a lot. It's also responsive so if you dont want this it will still maintain the aspect ratio with a half split.

![Screen Shot 2022-08-05 at 3 35 42 PM](https://user-images.githubusercontent.com/7501413/183148907-65e30569-8556-4d89-976d-339d1834ff6c.png)
![Screen Shot 2022-08-05 at 3 36 11 PM](https://user-images.githubusercontent.com/7501413/183148918-2c4449a9-257b-4a70-8430-a1b73ca58a92.png)

I think what will help is having a standard size for card image files, but for now this will certainly help with the ratio. Also, people like Chris who take photos using the rule of 1/3's will lose some of the image because it crops the sides off to maintain the ratio. Should keep it to a nice centered profile shot like Brents.
